content/docs/user-guide/testing/getting-started/_index.md

```diff
@@ -0,0 +1,233 @@
+
+Tests are written using standard [GoogleTest](https://github.com/google/googletest/blob/main/docs/index.md) syntax, which helps you write small functions to test your code. To pull in everything from GoogleTest plus a few convienient tools, add the following statement to your file:
+
+```cpp
+#include <AzTest/AzTest.h>
+```
+
+To keep test functions legible at a glance, we recommend using the [Osherove Naming Convention](https://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html) of `UnitOfWork_StateUnderTest_ExpectedBehavior`. This can be especially helpful when reading a report with many individual test case failures. Note that while Google recommends [not using any underscores](http://google.github.io/googletest/faq.html#why-should-test-suite-names-and-test-names-not-contain-underscore), tests should function normally as long as test and fixture names never start or end with an underscore. In a short a C++ example:
+
+```cpp
+TEST_F(Matrix4x4Tests, MatrixMultiply_InverseMatrix_ReturnIdentityMatrix)
+{
+    // (function under test would be called here)
+    ASSERT_TRUE(someErrorState);
+    EXPECT_TRUE(someResult);
+    EXPECT_FALSE(secondaryProperty);
+}
+```
+
+For more information on writing tests in C++, see [Using AzTest](/docs/user-guide/testing/aztest/aztest/).
+
+## GoogleBenchmark
+
+For performance benchmarks of small pieces of C++ code O3DE uses [GoogleBenchmark](https://github.com/google/benchmark/blob/main/docs/index.md), which is quite similar to GoogleTest. However, the definition of failure is a major difference between a normal test and a benchmark. In most tests a pass/fail status is directly evaluated to a boolean state, creating an objective report of success and failure. Benchmarks instead create a subjective performance metric. These metrics are most valuable when periodically recorded, to help detect trends over time. The only objective failure during a benchmark occurs when the code fails to run or crashes.
+
+The steps for configuring a GoogleBenchmark library are identical to the [steps above for GoogleTest libraries](#googletest), with a couple exceptions. The include statement is `#include <benchmark/benchmark.h>` and the cmake helper function is:
+
+```
+ly_add_googlebenchmark(
+    NAME MyModule.Benchmarks
+)
+```
+
+## PyTest
+
+Certain tests are easier to write in a scripting language like Python. The scope of these tests are typically at the [integration level](https://softwaretestingfundamentals.com/integration-testing/), verifying mid-to-high level functionality of workflows. This type of test helps verify system correctness similar to how an end-user will experience the software. Despite that positive aspect, these tests are typically slow and provide less specific failure information. Not all Python-based tests are integration tests: any Python-based tools will prefer unit tests written in Python. For all tests written in Python, O3DE uses [PyTest](https://docs.pytest.org/).
```

Comment:
```suggestion
Some tests are easier to write in Python, such as unit tests for Python-based tools or [integration tests](https://softwaretestingfundamentals.com/integration-testing/). For tests written in Python, O3DE uses [PyTest](https://docs.pytest.org/). Integration tests verify mid-to-high level functionality of workflows. It verifies the system correctness, similar to how an end-use experiences the software. Despite that, integration tests are typically slow, and they provide less specific failure information.
```

- Remove "in a scripting language like..." if we are only talking about tests written in Python. If the developer can write tests with other scripting languages and use PyTest with them, you can clarify that in another sentence. 
- Favor "some" over "certain", because certain as a determiner means having been determined but unspecified. When in this paragraph, you do specify at least some of the types of tests.
- Moved the last two sentences to the beginning of the paragraph since they are more general and introductory-level statements. 
- **Can you verify** if it's accurate to call the tests "integration test"? Specifying "integration test" throughout the paragraph helps clarify what tests you're talking about. Otherwise you could be talking about integration tests or tests written in Python.

Reply:
The first sentence is attempting to introduce a general testing concept, and then clarify that Python + PyTest is used in O3DE. Debating on whether that is an important mental-bridge to cross, or necessarily gets in the way as you describe.

Will clarify language around test-types.